docs/source/managed-federation/uplink.mdx

```diff
@@ -0,0 +1,90 @@
+---
+title: Apollo Uplink
+sidebar_title: Uplink
+---
+
+When using [managed federation](./overview/), your federated gateway regularly polls an endpoint called **Apollo Uplink** for its latest supergraph schema and other configuration:
+
+```mermaid
+graph LR;
+  subgraph "Your infrastructure"
+  serviceA[Products subgraph];
+  serviceB[Reviews subgraph];
+  gateway([Gateway]);
+  end
+  subgraph "Apollo cloud"
+  registry{{Apollo Schema Registry}};
+  uplink{{Apollo Uplink}}
+  end
+  serviceA & serviceB -->|Pushes schema| registry;
+  registry -->|Updates config| uplink;
+  gateway -->|Polls for config changes| uplink;
+  class registry secondary;
+  class uplink secondary;
+```
+
+To maximize uptime, Uplink is hosted simultaneously at _two_ endpoints, one in GCP and one in AWS:
+
+* GCP: `https://uplink.api.apollographql.com/`
+* AWS: `https://aws.uplink.api.apollographql.com/`
+
+By default in versions of `@apollo/gateway` vTODO and later, your gateway polls the GCP URL at a regular interval. Whenever a poll request fails, the gateway retries the request with the AWS URL. It continues swapping between the two URLs until the request succeeds, or until reaching the maximum number of retries.
```

Comment:
I believe this will be 0.45.0, but @trevor-scheer  can correct me!

Reply:
Correct!